packages/web-components/@groupby/elements-products/src/products-sayt.ts

```diff
@@ -39,15 +50,35 @@ export default class ProductsSayt extends ProductsBase {
   disconnectedCallback(): void {
     super.disconnectedCallback();
 
-    window.removeEventListener(SAYT_PRODUCTS_RESPONSE, this.setProductsFromEvent);
+    window.removeEventListener(SAYT_PRODUCTS_RESPONSE, this.setProductsFromProductsEvent);
+    window.removeEventListener(this.cacheResponseEventName, this.setProductsFromCacheEvent);
+  }
+
+  /**
+   * A string intended to be used as the name of the return event in
+   * cache requests for this component.
+   */
+  get cacheResponseEventName(): string {
+    return this.getCacheResponseEventName('products-sayt', this.componentId);
```

Comment:
@jmbtutor this is the line I am confused about. This is how I interpreted your suggestion to make getters, however maybe there is a better way to do this?

Reply:
done